bindings/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+option(FRAMEWORK_USES_PYTHON "Do you want to create the Python bindings" FALSE)
+
+if(FRAMEWORK_USES_PYTHON)
+
+    # Find the necessary Python libraries
+    find_package(Python3 COMPONENTS Interpreter Development REQUIRED)
+    message(STATUS "Using Python: ${Python3_EXECUTABLE}")
+
+    # Install the resulting Python package for the active interpreter
+    if(NOT PYTHON_INSTDIR)
+        set(PYTHON_INSTDIR ${Python3_SITELIB}/bipedal_locomotion)
+    endif()
+
+    # Find pybind11
+    find_package(pybind11 CONFIG REQUIRED)
```

Comment:
I would move all the ``find_packages`` in https://github.com/dic-iit/bipedal-locomotion-framework/blob/master/cmake/BipedalLocomotionFrameworkFindDependencies.cmake

Reply:
Sure!

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/compare/0f06adbbd29828e2f1d5d30e236a613c3a0f122a..f9ffcd82f625a9300d0d000fe236aae597707d69.

Checking the CMake macro, I'm a bit puzzled about which of the variables I should use, let me know if `FRAMEWORK_USE_pybind11` is not the intended choice.